update to PR (#6)
FelipeSantillan/Trixi.jl: docs/src/visualization.md

julia> savefig("solution-overview.png")
```

> In Trixi there are two different types of plots: 2D heatmap plots and 1D line plots.
> If you use `plot(sol)` Trixi will automatically choose the plot type, that fits the dimensions
> of the `sol` input.

Reviewer: ```suggestion
In Trixi, two plot types are available: 2D heatmap plots and 1D line plots.
If you use `plot(sol)`, Trixi will automatically choose the plot type that fits the dimensions
of the `sol` input: 2D/3D data will be visualized as a heatmap, 1D data as a line plot.
```

![plot-rho-uniform-mesh](https://user-images.githubusercontent.com/3637659/112101404-e0f64500-8ba6-11eb-9516-ad910c6813b2.png)

### Plotting a 3D solution as a 2D plot
It is possible to plot 2D slices from 3D simulation data with the same commands
as above. By default, plotting `sol` or creating a `PlotData2D` object from
a 3D simulation will create a 2D slice of the solution in the xy-plane. You can
customize this behavior by creating a `PlotData2D` object and passing
appropriate keyword arguments to `PlotData2D`:
* `slice` specifies the plane which is being sliced and can be `:xy`, `:xz`,
  or `:yz` (default: `:xy`)
* `point` specifies a three dimensional point. The sliced plane is then created

>   in such a way, that it lies on the point. (default: `(0.0, 0.0, 0.0)`)

Reviewer: ```suggestion
  in such a way, that it lies on the point. (default: `(0.0, 0.0, 0.0)`).
```

Any[]

julia> for x in range(0, stop=pi/2, length=6)
         pd = PlotData2D(sol, slice=:yz, point=(x, 0.0, 0.0))
         push!(plots, plot(pd["v1"], clims=(-1,1), title="x = "*string(round(x, digits=2))))
       end

julia> plot(plots..., layout=(2, 3), size=(750,350))
```
which results in a 2x3 grid of slices of the yz-plane:

![plot-v1-0.0-to-0.5pi](https://user-images.githubusercontent.com/3637659/102917883-417dc500-4486-11eb-9bd3-d18efd9c8337.png)

### Creating a 1D plot via a PlotData1D object
When plotting a 1D solution with `plot(sol)` Trixi automatically creates a [`PlotData1D`](@ref) object.
If you want to customize your 1D plot you can create a [`PlotData1D`](@ref) object manually as follows:
```julia
julia> pd = PlotData1D(sol)
```
In a very similar fashion to [`PlotData2D`](@ref), you can customize your plot:
* `plot(pd)` creates a plot as on default.
* `plot(pd["rho", "p"])` only plot specific variables. In this case `rho` and `p`.
* `plot!(getmesh(pd))` adds mesh lines after creating a plot.

> * Any attributes from [Plots](https://docs.juliaplots.org/latest/) can be used.
>   (ex. `plot(pd, yguide=:temperature)`)

Reviewer: ```suggestion
* Any attributes from [Plots](https://docs.juliaplots.org/latest/) can be used, e.g., `plot(pd, yguide=:temperature)`.
```

> * `pd = PlotData2D(adapt_to_mesh_level(sol, 4)...)` refines the mesh before plotting;
>   (in this example to a mesh with level 4)

Reviewer: ```suggestion
* `pd = PlotData1D(adapt_to_mesh_level(sol, 4)...)` refines the mesh before plotting;
  (in this example to a mesh with level 4)
```
Have you actually tested whether this works?

Author: Yes, this will work.

You can also customize the [`PlotData1D`](@ref) object itself by passing attributes
to the [`PlotData1D`](@ref) constructor:
* `solution_variables` specifies the variables to be plotted.
* `nvisnodes` sets the amount of nodes per element which the solution then is interpolated on.

> ### Plotting a 2D solutions as a 1D plot

Reviewer: Shouldn't we also mention 3D to 1D? I don't it requires a new section, but maybe change this section to 
```suggestion
### Plotting 2D/3D solutions as a 1D plot
```
and mention in the beginning that everything here is discussed for 2D but applies for 3D as well, with the only difference being that the `point` argument must have 3 dimensions.

Author: Done in 06cbcce.
